frontend/src/components/pagination/index.tsx

```diff
@@ -0,0 +1,76 @@
+import * as React from 'react';
+import classnames from 'classnames';
+
+import * as styles from './style.css';
+
+type Props = {
+  currentPage: number;
+  totalItems: number;
+  itemsPerPage: number;
+  onPageChange?: (nextPage: number) => void;
+};
+
+export class Pagination extends React.Component<Props, {}> {
+  private changePage(nextPage: number, e: React.MouseEvent<HTMLSpanElement>) {
+    if (nextPage < 0 || nextPage >= this.getTotalPages()) {
+      return;
+    }
+
+    if (this.props.onPageChange) {
+      this.props.onPageChange(nextPage);
+    }
+  }
+
+  private getTotalPages() {
+    const { totalItems, itemsPerPage } = this.props;
+    return Math.floor(totalItems / itemsPerPage);
+  }
+
+  public render() {
+    const { currentPage } = this.props;
+    const totalPages = this.getTotalPages();
+
+    if (totalPages === 0) {
+      return null;
+    }
+
+    const prevClasses = classnames(styles.action, {
+      [styles.disabled]: currentPage === 0,
+    });
+
+    const nextClasses = classnames(styles.action, {
+      [styles.disabled]: currentPage >= totalPages - 1,
+    });
+
+    return (
+      <div className={styles.pagination}>
+        <span
+          className={prevClasses}
+          onClick={this.changePage.bind(this, currentPage - 1)}
+        >
+          Prev
+        </span>
+        {Array(totalPages)
+          .fill(null)
+          .map((_, i) => i + 1)
```

Comment:
I'd put this in a utility method, mainly to make it more readable :)

`range(totalPages).map()`